react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,212 @@
+/* @flow */
+
+import React from 'react';
+
+declare var APP: Object;
+declare var JitsiMeetJS: Object;
+
+/**
+ * All toolbar buttons' descriptions.
+ */
+export default {
+    'microphone': {
+        id: 'toolbar_button_mute',
```

Comment:
I do not approve of the id that is a magic string because it's error prone. Search for the string `'toobar_button_mute'` and you'll find it 4 times. When you combine it with my other comment about properties with values functions and how they are to be written, and you comply with camel case for function names afterwards, I'd like you to think of an elegant way to use functions/function names instead of magic numbers.